main/lsp/LSPTypechecker.cc

```diff
@@ -438,6 +439,33 @@ LSPQueryResult LSPTypechecker::query(const core::lsp::Query &q, const std::vecto
     return LSPQueryResult{move(out.second)};
 }
 
+vector<core::LocalVariable> LSPTypechecker::localsForMethod(const core::SymbolRef method) const {
```

Comment:
I'm not sure if it belongs here.
cc @jvilk-stripe for decision power on what goes into `LSPTypechecker` and what doesn't.

Reply:
Thanks for the ping. Like `getReferencesToSymbol`, niche queries like this one should go on LSPLoop:

https://github.com/sorbet/sorbet/blob/master/main/lsp/lsp.h#L76

I'd like for LSPTypechecker to only surface general-purpose building block methods.

Would this make sense as a new type of query?

https://github.com/sorbet/sorbet/blob/master/main/lsp/LSPTypechecker.cc#L410

Reply:
See 9074281